internal/core/command/v2/mockclients/interfaces/device.go

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package interfaces
+
+import "github.com/edgexfoundry/go-mod-core-contracts/v2/v2/clients/interfaces"
+
+type MockDeviceClient interface {
```

Comment:
Why are these needed. Mockery should create all that is required.

Reply:
The V2 client library interfaces are defined under go-mod-core-contracts, and Mockery doesn't directly support the feature to generate mocks from an external go modules.  My trick is to create corresponding Mock interfaces with inline fields directly referring to the V2 client library interface of go-mod-core-contracts, so that Mockery could generate the mocks within edgex-go.  However, as mocks have been moved to go-mod-core-contracts, I'll remove these mocking interfaces.